metadata.schema.json

```diff
@@ -0,0 +1,51 @@
+{
+    "type": "object",
+    "additionalProperties": {
+      "type": "object",
+      "properties": {
+        "title": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "viewType": {
+          "type": "string",
+          "enum": ["functionSnippetWithNumberedGuide", "functionSnippet", "triggerMatch", "triggerProject"]
+        },
+        "guide": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "title": {
+                "type": "string"
+              },
+              "body": {
+                "type": "string"
+              }
+            },
+            "required": ["title"],
+            "additionalProperties": false
+          },
+          "docUrl": {
```

Comment:
[q] is the docUrl here part of the guide object? I was thinking it was on the same level as the guide object.